app/views/footer.scala.html

```diff
@@ -0,0 +1,18 @@
+@import views.html.bootstrap._
+@import models.user.VersionTable
+
+@(showVersion: Boolean = false)
```

Comment:
The lack of a version number in the footer on the validation and rapid validation pages was actually an oversight, we would like to have the version number everywhere! I think this is a great solution to the fix, but could you remove this functionality so that the version num is everywhere? :grin: